podcasts/IconSelectorCell.swift

```diff
@@ -191,7 +199,7 @@ class IconSelectorCell: ThemeableCell, UICollectionViewDataSource, UICollectionV
     }
 
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        13
+        14
```

Comment:
A NitPick suggestion in case we decide to rotate out future holiday-specific images. 

```suggestion
IconType.allCases.count
```

Reply:
Def not a nitpick and way better, done in df94a0056e420cebfa4df1c76688d82c5f2edd05